src/modules/toast/toasts/meta-transactions.tsx

```diff
@@ -76,7 +76,21 @@ export function getHighCongestionErrorToast(): Omit<Toast, 'id'> {
     title: t('@dapps.toasts.meta_transactions.high_congestion_error.title'),
     body: (
       <MetaTransactionError
-        text={t('@dapps.toasts.meta_transactions.high_congestion_error.body')}
+        text={
+          <span>
+            {t('@dapps.toasts.meta_transactions.high_congestion_error.body', {
+              a: (text: string) => (
+                <a
+                  href="https://polygonscan.com/gastracker"
```

Comment:
Shouldn't this discriminate also mumbai in case we are using testnet? might be unnecessary but doesn't feel right to use polygonscan for testnet